C_CPP/OOPs in C++/Abstract class in C++/Abstract_class.md

```diff
@@ -0,0 +1,74 @@
+# Abstract Class in C++
+▶️ Abstract class is a class whose instantiation is restricted. 
```

Comment:
Using such types of colored emojis is not preferred, remove them and use simple bullet points.

Reply:
@deeqakkk Updated the required changes.